src/experiments/kfold_opportunity.py

```diff
@@ -0,0 +1,96 @@
+"""
+test with new config
+
+"""
+
+import os
+import random
+
+import numpy as np
+from sklearn.model_selection import KFold
+
+import utils.settings as settings
+from evaluation.metrics import accuracy
+from evaluation.conf_matrix import create_conf_matrix
+from loader.Preprocessor import Preprocessor
+from models.JensModel import JensModel
+from utils.filter_activities import filter_activities
+from utils.folder_operations import new_saved_experiment_folder
+from utils.DataConfig import OpportunityConfig
+from utils.save_all_recordings import save_all_recordings, load_all_recordings
+
+
+
+# Init
+data_config = OpportunityConfig(dataset_path='data/opportunity-dataset')
+settings.init(data_config)
+random.seed(1678978086101)
+
+# Load dataset
+recordings = settings.DATA_CONFIG.load_dataset()
+
+# Preprocess
+recordings = Preprocessor().jens_preprocess(recordings)
+
+# # Save recordings - Kirill will help you to save some time
+# path_preprocessed_folder = 'data/preprocessed_recordings'
+# preprocessing_file_name = 'all_recordings'
+# save_all_recordings(recordings, path_preprocessed_folder, preprocessing_file_name)
+# recordings = load_all_recordings(os.path.join(path_preprocessed_folder, preprocessing_file_name))
+
+# KFold cross validation (leave recordings out)
+k = 5
+accuracies: "list[float]" = []
+recordings: np.ndarray = np.array(recordings)
+
+window_size = 100
+n_features = recordings[0].sensor_frame.shape[1]
+n_outputs = settings.DATA_CONFIG.n_activities()
+
+# Create Folder, save model export and evaluations there
+experiment_folder_path = new_saved_experiment_folder(
+    "template_opportunity_exp"
+)
+
+k_fold = KFold(n_splits=k, random_state=None)
+recordings_subject_str = []
+for idx, (train_index, test_index) in enumerate(k_fold.split(recordings)):
+    print(f"Doing fold {idx}/{k-1} ... ============================================================")
+    recordings_train, recordings_test = (
+        recordings[train_index],
+        recordings[test_index],
+    )
+    to_recordings_subject_str = lambda recordings: ''.join(list(map(lambda recording: recording.subject, recordings)))
+    recordings_subject_str.append(f"recordings_subjects: train: {to_recordings_subject_str(recordings_train)}, test: {to_recordings_subject_str(recordings_test)}")
+
+    model = JensModel(
+        n_epochs=5,
+        window_size=100,
```

Comment:
```suggestion
        window_size=window_size,
```

same as in the other file